test/test_tools.rb

```diff
@@ -154,10 +154,52 @@ def test_add_rpath_fat
   end
 
   def test_delete_rpath
-    pass
+    groups = SINGLE_ARCHES.map do |arch|
+      ["hello.bin", "hello_actual.bin"].map do |fn|
+        fixture(arch, fn)
+      end
+    end
+
+    groups.each do |filename, actual|
+      FileUtils.cp filename, actual
+      MachO::Tools.delete_rpath(actual, "made_up_path")
+
+      original = MachO::MachOFile.new(filename)
+      modified = MachO::MachOFile.new(actual)
+
+      assert modified.ncmds < original.ncmds
+      assert modified.sizeofcmds < original.sizeofcmds
+      assert modified.rpaths.size < original.rpaths.size
```

Comment:
Maybe add a check that the file size of the original and modified file is still the same? (None of the modifications should alter the file size.)